src/base/utils/string.cpp

```diff
@@ -42,136 +40,69 @@
 #include <QRegExp>
 #endif
 
-#if defined(Q_OS_MACOS) || defined(__MINGW32__)
-#define QBT_USES_QTHREADSTORAGE
-#include <QThreadStorage>
-#endif
-
-namespace
+#ifndef QBT_USE_QCOLLATOR
+int Utils::String::naturalCompare(const QString &left, const QString &right, const Qt::CaseSensitivity caseSensitivity)
```

Comment:
Did you change this algorithm or just transfer existing logic?

Reply:
Only transferred.